frontend/src/layouts/ContentLayout.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { CssBaseline, Paper } from '@mui/material';
+
+const styles = {
+  main: {
+    width: 'auto',
+    display: 'block', // Fix IE 11 issue.
+    marginLeft: 100,
+    marginRight: 100,
+  },
+  paper: {
+    marginTop: 3,
+    paddingBottom: 20,
+    flexDirection: 'column',
+    alignItems: 'center',
+    p: 2,
+    width: 900,
+    marginLeft: "auto",
+    marginRight: "auto",
+  },
+};
+
+const ContentLayout =({children}) =>{
+  
+  return (
+    <main SX={styles.main}>
```

Comment:
This prints error on console, should be changed with lowercase `sx`

Reply:
This was fixed in #442.